src/mobility/components/OperatorBenefit.tsx

```diff
@@ -56,15 +59,70 @@ export const OperatorBenefit = ({
   );
 };
 
+const BenefitImageAsset = ({formFactor} : {formFactor: FormFactor}) => {
+  switch (formFactor) {
+    case FormFactor.Car:
+      return <BundlingCarSharing />;
+    case FormFactor.Bicycle:
+      return <BundlingCityBike />;
+    default:
+      return <View />;
+  }
+}
+
+type BenefitImageProps = {
+  formFactor: FormFactor;
+  eligible: boolean;
+  style?: ViewStyle;
+};
+
+const BenefitImage = ({
+  formFactor,
+  eligible,
+  style,
+}: BenefitImageProps): JSX.Element => {
+
+  return (
+    <View style={style}>
+      <>
+        <BenefitImageAsset formFactor={formFactor} />
+        {eligible && (<BenfitEligibilityIndicator />)}
+      </>
+    </View>
+  );
+};
+
+const BenfitEligibilityIndicator = () => {
+  const styles = useStyles();
+
+  return (
+    <View style={styles.indicator}>
+      <Check width={24} height={24} />
+    </View>
+  );
+};
+
+
 const useStyles = StyleSheet.createThemeHook((theme) => ({
   benefitContainer: {
     flexDirection: 'row',
+    alignItems: 'center',
+  },
+  benefitContent: {
+    flex: 4,
+    marginStart: theme.spacings.medium,
   },
   benefitImage: {
+    flex: 1,
+    resizeMode: 'contain',
     marginRight: theme.spacings.medium,
-    justifyContent: 'center',
   },
-  benefitContent: {
-    flex: 1,
+  indicator: {
+    position: 'absolute',
+    right: 0,
+    top: 0,
+    borderRadius: theme.border.radius.circle,
```

Comment:
You are right, I only need the `right` properties because otherwise it will show like this 
<img width="108" alt="image" src="https://github.com/AtB-AS/mittatb-app/assets/1777333/588d5b1c-614c-424f-bb95-f4b527cb3c0c">

If I added the `right` properties it will show correctly as shown in Figma :
<img width="103" alt="image" src="https://github.com/AtB-AS/mittatb-app/assets/1777333/d46999d9-cd7d-4ec0-ac4d-39dd9e434989">

I will remove the unnecessary properties